### PR TITLE
000 update home page

### PR DESCRIPTION
**Pull request by glup100**
Updates to the index.html and added the images used to the images folder

### CODE REVIEW

File: views/index.html [hunk 1]
```diff
@@ -1,18 +1,37 @@
...
-                <a href="/hello-world">Hello World</a>
-            </li>
-            <li>
-                <a href="/job-roles">Job Roles</a>
-            </li>
-        </ul>
+        <div class="display-cards">
+            <div class="card">
+                <img src="/images/CardImage1.png"
+                    class="card-img-top" alt="...">
```
Comment (glenn-hamilton) [inline views/index.html]: This isn't very useful alt text

File: views/index.html [hunk 2]
```diff
@@ -1,18 +1,37 @@
...
+                    <h5 class="card-title">Job Roles</h5>
+                    <p class="card-text">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quam exercitationem
+                        temporibus culpa a cumque quisquam debitis quod atque iste quasi!</p>
+                    <a href="/job-roles" class="btn btn-primary">Job Roles</a>
+                </div>
+            </div>
 
+            <div class="card">
+                <img src="/images/CardImage2.png"
+                    class="card-img-top" alt="...">
```
Comment (glenn-hamilton) [inline views/index.html]: This isn't very useful alt text

File: views/index.html [hunk 3]
```diff
@@ -1,18 +1,37 @@
...
+                <div class="card-body">
+                    <h5 class="card-title">Log in</h5>
+                    <p class="card-text">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quam exercitationem
+                        temporibus culpa a cumque quisquam debitis quod atque iste quasi!</p>
+                    <a href="/login" class="btn btn-primary">Login</a>
```
Comment (glenn-hamilton) [inline views/index.html]: This will show up even if I'm logged in

**Review by glenn-hamilton — APPROVED** [review]

**Review by DarraghHaughey-Kainos — APPROVED** [review]
Lovely :)

**Pull request merged**